packages/documentation/src/components/PlaygroundV2/utils.ts

```diff
@@ -13,6 +13,33 @@ import { themeSwitcher } from '@siemens/ix';
 
 const repositoryUrl = 'https://github.com/siemens/ix/tree/main/packages';
 
+export function getBranchPath(framework: TargetFramework) {
+  let path = 'html';
+
+  const branch = 'main';
+
+  if (framework === TargetFramework.ANGULAR) {
+    path = 'angular';
+  }
+
+  if (framework === TargetFramework.REACT) {
+    path = 'react';
+  }
+
+  if (framework === TargetFramework.VUE) {
+    path = 'vue';
+  }
```

Comment:
Eventually would be more clearly?
  switch (framework) {
    case TargetFramework.ANGULAR:
      path = 'angular';
      break;
    case TargetFramework.REACT:
      path = 'react';
      break;
    case TargetFramework.VUE:
      path = 'vue';
  }

Reply:
No if statements are fine imo